libs/pika/command_line_handling/src/parse_command_line.cpp

```diff
@@ -338,6 +338,11 @@ namespace pika::detail {
                 "allowed values: 0 - no NUMA sensitivity, 1 - allow only for "
                 "boundary cores to steal across NUMA domains, 2 - "
                 "no cross boundary stealing is allowed (default value: 0)")
+#if defined(PIKA_HAVE_MPI)
+            ("pika:mpi-completion-mode", value<std::size_t>(),
+                "the pika MPI polling completion mode (only available if MPI "
+                "built with MPI support)")
+#endif
```

Comment:
Really would like the ability for modules to add options in a modular way - then I would not have used ENV vars at all.

Reply:
I understand that. I think for real-world use cases the environment variables are still the most useful way of setting options since command line options might never make it into pika.

Note, this is likely anyway a temporary change until we get something in the direction of https://github.com/pika-org/pika/issues/1144.

Ideally, users don't have to set the completion mode at all (because the default is good enough for most use cases).